### PR TITLE
Teaser Launch Site Updates

### PR DESCRIPTION
**Pull request by DavidKellyAD**
• Removed unused code from the device detection js • Resolved some spacing on the download button for mobile • Updated the press release to embargo until Jan 31st for press related testers • Hid app store links

### CODE REVIEW

File: js/device-detection.js [hunk 1]
```diff
@@ -1,27 +1,3 @@
...
-
-    // iOS detection from: http://stackoverflow.com/a/9039885/177710
-    if (/iPad|iPhone|iPod/.test(userAgent) && !window.MSStream) {
-        return "iOS";
-    }
-
-    return "unknown";
-}
-
- */
```
Comment (instructedmachines) [inline js/device-detection.js:24]: Why did you remove this? It looks like this should work

**Review by instructedmachines — APPROVED** [review]

File: js/device-detection.js (hunk 1 above)
Comment (DavidKellyAD) [reply js/device-detection.js:24]: It should but halfway through doing this I decided it this was a great opportunity to kick the tires on Chat GPT's coding ability. This didn't connect to anything so it was just taking up space.
